### DISCUSSION TITLE
캐릭터 크기를 줄이거나 키울 수 있을까요??

### DISCUSSION THREAD

**Discussion by HRaaaaa** [naverz/zepeto-studio-kor#1105] [5. Q&A]
캐릭터가 미니어처의 포토존으로 가면 (상호작용키) 포토존의 맞는 크기로 줄어들고
포토존을 다 즐긴 후 다시 원상태로 돌아 왔으면 합니다. 

개발로 가능한 부분인가요?

**Comment by zlmrzlo** [comment]
개발 가능한 부분으로 생각합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 플레이어의 localScale 값을 수정해서 크기 변경이 가능합니다.

예) 2배로 커지게 하기 
ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.gameObject.transform.localScale = new Vector3(2,2,2);

예) 0.5배로 작아지게 하기 
ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.gameObject.transform.localScale = new Vector3(0.5,0.5,0.5);

**Reply by chlguswls32** [reply]
local Scale변경으로 재미있는 기능들도 만들수있겠네요 ㅎㅎ
